### PR TITLE
draft outline for all episodes

### PR DESCRIPTION
**Pull request by tobyhodges**
This provides the questions, objectives, and key points for all of the proposed episodes.

There are also a few discussion questions, instructor notes, homework tasks, etc that I wanted to note down while they were fresh in my mind. Hat-tip to @sstevens2 for the proposed discussion question in `iterative-development.md`.

When reviewing, please suggest improvements including wholesale removal and addition of objectives, questions, etc - this is a first draft and should be considered fair game for drastic revision.

### CODE REVIEW

File: episodes/backwards-design.md [hunk 1]
```diff
@@ -1,101 +1,83 @@
...
 
-## Challenge 2: how do you nest solutions within challenge blocks?
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-:::::::::::::::::::::::: solution 
+## Introductions
 
-You can add a line with at least three colons and a `solution` tag.
+The first discussion session will begin with a full round of Introductions,
+where each trainee will be asked to share some details about themselves
```
Comment (sstevens2) [inline episodes/backwards-design.md:40]: folks leading this session should probably introduce themselves as well?

File: episodes/collaboration.md [hunk 2]
```diff
@@ -6,96 +6,53 @@ exercises: 2
...
 
-You can add a line with at least three colons and a `solution` tag.
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-:::::::::::::::::::::::::::::::::
-::::::::::::::::::::::::::::::::::::::::::::::::
+## Discussion Questions
 
-## Figures
+- What decision-making framework(s) have you used in your previous collaborations?
```
Comment (sstevens2) [inline episodes/collaboration.md:37]: Might broaden this a bit as folks may not have experience with any specific decision-making frameworks. 
```suggestion
- How have you made decisions with collaborators in the past?
- Did you use any formal decision-making framework(s)?
- What might be the advantages and drawbacks of using a formal decision-making framework?
```
Also added a pro/con discussion question - sorry for combining suggestions

Comment (sstevens2) [reply episodes/collaboration.md:37]: Maybe a discussion of working alone vs collaboration pros and cons would be a good addition based on the objectives for this lesson.

File: episodes/introduction.md [hunk 3]
```diff
@@ -1,102 +1,95 @@
...
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
+## Curriculum is a work in progress
 
-:::::::::::::::::::::::: solution 
+Any lesson or curriculum has room for improvement,
+especially when it is relatively new and not extensively tested.
+The CLDT curriculum is still in the early stages of the development process:
+it has only been taught twice, both times by its primary authors.
```
Comment (sstevens2) [inline episodes/introduction.md:49]: like this here but slight concern about forgetting to update it

File: episodes/introduction.md [hunk 4]
```diff
@@ -1,102 +1,95 @@
 ---
-title: "Using Markdown"
-teaching: 10
-exercises: 2
+title: "Welcome"
```
Comment (sstevens2) [inline episodes/introduction.md:2]: think this episode as a whole will probably need more text about the reading each week and discussion sessions to set the stage for how trainer training will run.

File: episodes/introduction.md [hunk 5]
```diff
@@ -1,102 +1,95 @@
...
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+## Introductions
```
Comment (sstevens2) [inline episodes/introduction.md:33]: Might it be better to move intros here or will this episode not be a discussion itself?

File: episodes/operations.md [hunk 6]
```diff
@@ -6,96 +6,61 @@ exercises: 2
...
-![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
+## Trainer Checkout
 
-## Math
+Do the following to complete your Trainer certification:
 
-One of our episodes contains $\LaTeX$ equations when describing how to create
-dynamic reports with {knitr}, so we now use mathjax to describe this:
+1. Submit a pull request to fix [an issue on the Collaborative Lesson Development Training curriculum][cldt-issues].
+2. Teach Collaborative Lesson Development Training alongside an experienced Trainer.
```
Comment (sstevens2) [inline episodes/operations.md:52]: Do you think they would need to teach all parts? Or could they teach in sections 1-4 or 5-6 to complete trainer certification?

File: episodes/backwards-design.md (hunk 1 above)
Comment (tobyhodges) [reply episodes/backwards-design.md:40]:
```suggestion
where trainees and facilitators will be given the opportunity to share some details about themselves
```

File: episodes/introduction.md [hunk 7]
```diff
@@ -1,102 +1,95 @@
...
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+By the end of this session, participants should be able to...
+
+- describe the format of the Trainer Training program and sessions.
+- summarise the lesson development experience of the participants in the cohort.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
```
Comment (tobyhodges) [inline episodes/introduction.md:22]:
```suggestion
FIXME: Add more text about the reading each week and discussion sessions to set the stage for how trainer training will run.
```

File: episodes/lesson-infrastructure.md [hunk 8]
```diff
@@ -6,96 +6,68 @@ exercises: 2
...
- 
-```output
-[1] "This new lesson looks good"
-```
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-:::::::::::::::::::::::::::::::::
+## Discussion Questions
 
+- FIXME
```
[the remark this answers is not in the document]
Comment (tobyhodges) [reply episodes/lesson-infrastructure.md:39]:
```suggestion
- a discussion of anticipated sticking points for learners who may not be familiar with git/GitHub/the workbench? 
- how to support trainees with these sticking points.
```

File: episodes/introduction.md (hunk 5 above)
Comment (tobyhodges) [reply episodes/introduction.md:33]: resolved in cf2666a

File: episodes/introduction.md (hunk 3 above)
Comment (tobyhodges) [reply episodes/introduction.md]: resolved in cf2666a

**Pull request merged**